File: Week2/Code/boilerplate.py
#!/usr/bin/env python3
"""Prints This is a boilerplate."""
__appname__='simple boilerplate'
__author__='Ruth Keane (ruth.keane19@imperial.ac.uk)'
__version__='0.0.1'
__license__="License for this code/programme"
#Desc: prints "This is a boilerplate"
#Arguments: none
#Input: none
#Output: none
#Date: Oct 2019
## imports##
import sys #module to interface our program with the operating system
##constants##

##functions##
def main(argv):
    """Main entry point of the program"""
    print('This is a boilerplate') # Note: indented using 2 tabs or 4 space
    return 0

if __name__=="__main__":
    """Makes sure the "main" function is called from command line"""
    status = main(sys.argv)
    sys.exit(status)

File: Week3/Code/boilerplate.R
#Author: Ruth Keane (ruth.keane19@imperial.ac.uk)
#Script: boilerplate.R
#Desc: boilerplate script-containing function whih finds and prints the class of the arguments
#Arguments: 2 
#Input: for MyFunction, MyFunction(Arg1,Arg)
#Output: none
#Date: Oct 2019
MyFunction <-function(Arg1, Arg2){
        print(paste("Argument", as.character(Arg1), "is a", class(Arg1)))
        print(paste("Argument", as.character(Arg2), "is a", class(Arg2)))

        return(c(Arg1, Arg2))
}
MyFunction(1,2)
MyFunction("Riki","Tiki")